crates/aptos/src/node/local_testnet/processors.rs

```diff
@@ -128,10 +152,14 @@ impl ProcessorManager {
             // many in a localnet environment.
             db_pool_size: Some(8),
             gap_detection_batch_size: 50,
+            parquet_gap_detection_batch_size: 50,
             pb_channel_txn_chunk_size: 100_000,
             per_table_chunk_sizes: Default::default(),
-            transaction_filter: Default::default(),
             grpc_response_item_timeout_in_secs: 10,
+            transaction_filter: Default::default(),
+            deprecated_tables: Default::default(),
+            default_sleep_time_between_request:
+                IndexerGrpcProcessorConfig::default_sleep_time_between_request(),
```

Comment:
Please check if the added default field values are sensible.

Reply:
Yes, these are sensible.